ibm/service/vpc/resource_ibm_is_bare_metal_server_network_interface_allow_float.go

```diff
@@ -226,16 +226,36 @@ func createVlanTypeNetworkInterfaceAllowFloat(context context.Context, d *schema
 	}
 	nicOptions.InterfaceType = &interfaceType
 
-	if ais, ok := d.GetOk(isBareMetalServerNicAllowIPSpoofing); ok {
+	log.Printf("[INFO] UJJK isBareMetalServerNicAllowIPSpoofing GET %v", d.Get(isBareMetalServerNicAllowIPSpoofing))
+	//getok
+	ais, ok := d.GetOk(isBareMetalServerNicAllowIPSpoofing)
+	log.Printf("[INFO] UJJK isBareMetalServerNicAllowIPSpoofing GETOK %v, =====  %t", ais, ok)
+
+	//getokexiss
+	aisx, ok := d.GetOkExists(isBareMetalServerNicAllowIPSpoofing)
+	log.Printf("[INFO] UJJK isBareMetalServerNicAllowIPSpoofing GetOkExists %v, =====  %t", aisx, ok)
+
+	if ais, ok := d.GetOkExists(isBareMetalServerNicAllowIPSpoofing); ok {
 		allowIPSpoofing := ais.(bool)
 		if allowIPSpoofing {
 			nicOptions.AllowIPSpoofing = &allowIPSpoofing
 		}
 	}
-	if ein, ok := d.GetOk(isBareMetalServerNicEnableInfraNAT); ok {
+
+	log.Printf("[INFO] UJJK isBareMetalServerNicEnableInfraNAT GET %v", d.Get(isBareMetalServerNicEnableInfraNAT))
+	//getok
+	kais, ok := d.GetOk(isBareMetalServerNicEnableInfraNAT)
+	log.Printf("[INFO] UJJK isBareMetalServerNicEnableInfraNAT GETOK %v, =====  %t", kais, ok)
+
+	//getokexiss
+	kaisx, ok := d.GetOkExists(isBareMetalServerNicEnableInfraNAT)
+	log.Printf("[INFO] UJJK  isBareMetalServerNicEnableInfraNAT GetOkExists %v, =====  %t", kaisx, ok)
+
+	if ein, ok := d.GetOkExists(isBareMetalServerNicEnableInfraNAT); ok {
```

Comment:
Instead of using deprecated `d.GetOkExists` method, let's continue to use `d.GetOk` with additional checks in place.